src/pages/PlottingProgress.vue

```diff
@@ -214,21 +217,62 @@ export default defineComponent({
     },
     startPlotting() {
       this.plotting = true
-      this.fakeProgress()
-      timer = window.setInterval(() => (this.elapsedms += 100), 100)
+      this.plottingProgress()
     },
     pausePlotting() {
       this.plotting = false
-      clearInterval(interval)
       clearInterval(timer)
     },
-    async fakeProgress() {
-      interval = window.setInterval(() => {
-        this.plottingData.finishedGB += util.random(0, 50) / 40
-        console.log(this.plottingData)
-        if (this.plottingData.finishedGB >= this.allocatedGB)
-          this.pausePlotting()
-      }, util.random(200, 1000))
+    async plottingProgress() {
+      // If the local node is Syncing. Must wait until done.
+      let blockNumberData = await Promise.all([
+        this.client.getLocalLastBlockNumber(),
+        this.client.getNetworkLastBlockNumber(),
+      ])
+      do {
+        blockNumberData = await Promise.all([
+          this.client.getLocalLastBlockNumber(),
+          this.client.getNetworkLastBlockNumber(),
+        ])
+        this.plottingData.status = "Syncing node  " + blockNumberData[0].toLocaleString() + " of " + blockNumberData[1].toLocaleString() +" Blocks"
+        await new Promise((resolve) => setTimeout(resolve, 1500))
+      } while (blockNumberData[0] < blockNumberData[1])
+      
+      this.plottingData.status = lang.startingFarmer
+      // After local node is fully Synced, the farmer will be able to actualy plot and farm. 
+      const { publicKey, mnemonic } = await startFarming(this.plotDirectory.replace("/subspace.plot", ""))
+ 
+      // When farmer start, temporal mnemonic storage, when user accept on modal that mnemonic was backed up. We remove it from local storage
+      // Check SaveKeys.vue.
+      if (publicKey && mnemonic) {
+        // TODO: Fully remove LocalStorage. Find a secure way to pass mnemonic to SaveKeys modal.
+        LocalStorage.set("mnemonic", mnemonic)
```

Comment:
I couldn't see how mnemonic is removed from LocalStorage after here? 
I saw a reset on mnemonic on here: https://github.com/subspace/subspace-desktop/pull/46/files#diff-653e0382a5f2a0bcb258b7abb55fe7ac7e6e00e6e9295618f526671a54d11cc5R66

But I believe that code will be executed before here?